types/src/generator/type.ts

```diff
@@ -103,11 +103,22 @@ export function isUnsatisfiable(typeNode: ts.TypeNode) {
 
 // Strings to replace in fully-qualified structure names with nothing
 const replaceEmpty =
-  /^workerd::api::public_beta::|^workerd::api::|^workerd::jsg::|::|[ >]/g;
+  /^workerd::api::public_beta::|^workerd::api::node::|^workerd::api::gpu::|^workerd::api::|^workerd::jsg::|::|[ >]/g;
```

Comment:
adding some comments here and when to update this would be helpful. My fear is that this is a bit buried and obscure

Reply:
https://github.com/cloudflare/workerd/pull/2379/commits/409f10dca728a04a7800f20173a96aa75a4ae1b8